website/docs/addon_houdini_admin.md

```diff
@@ -126,6 +126,62 @@ Disable this if you want the publisher to ignore start and end handles specified
 When working in per-shot style you always publish data in context of current asset (shot). This validator checks if this is so. It is optional so it can be disabled when needed.
 
+--- 
+
+## Stock Product Types
+A list of stock product types that come with Ynput's Houdini addon.
+
+| Product Type Label | Product Type Name | Description | 
+|--|--|--|
+| `Arnold ASS` | `ass` | Arnold Scene Source .ass Archive. |
+| `Arnold ROP` | `arnold_rop` | Arnold Render. |
+| `Camera (Abc)` | `camera` |  Single baked camera from Alembic ROP. |
+| `Composite (Image Sequence)` | `imagesequence` | Composite ROP to Image Sequence. |
+| `Houdini Digital Asset (Hda)` | `hda` | Houdini Digital Asset file. |
+| `Karma ROP` | `karma_rop` | Karma Render. |
+| `Mantra IFD` | `mantraifd` | Mantra .ifd Archive. |
+| `Mantra ROP` | `mantra_rop` | Mantra Render. |
+| `PointCache (Abc)` | `pointcache` | pointcache as Alembic. |
+| `PointCache (Bgeo)` | `pointcache` | pointcache as BGEO. |
+| `Redshift ROP` | `redshift_rop` | Redshift Render. |
+| `Review` | `review` | Review with OpenGL ROP. |
+| `Static Mesh (FBX)` | `staticMesh` | Static Meshes as FBX. |
+| `USD (experimental)` | `usd` | Universal Scene Description. |
+| `USD Render (experimental)` | `usdrender` | USD Render. |
+| `VDB Cache` | `vbdcache` | OpenVDB from Geometry ROP. |
+| `VRay ROP` | `vray_rop` | VRay Render. |
+
+* **Product Type Label:** It's mostly used in AYON UI e.g. Publisher Tool.
+* **Product Type Name:** The name used in the background. AYON Settings expects you to use this name not the label.
+
+--- 
+
+## Stock Product Types
+A list of stock product types that come with Ynput's Houdini addon.
+
+| Product Type Label | Product Type Name | Description | 
+|--|--|--|
+| `Arnold ASS` | `ass` | Arnold Scene Source .ass Archive. |
+| `Arnold ROP` | `arnold_rop` | Arnold Render. |
+| `Camera (Abc)` | `camera` |  Single baked camera from Alembic ROP. |
+| `Composite (Image Sequence)` | `imagesequence` | Composite ROP to Image Sequence. |
+| `Houdini Digital Asset (Hda)` | `hda` | Houdini Digital Asset file. |
+| `Karma ROP` | `karma_rop` | Karma Render. |
+| `Mantra IFD` | `mantraifd` | Mantra .ifd Archive. |
+| `Mantra ROP` | `mantra_rop` | Mantra Render. |
+| `PointCache (Abc)` | `pointcache` | pointcache as Alembic. |
+| `PointCache (Bgeo)` | `pointcache` | pointcache as BGEO. |
+| `Redshift ROP` | `redshift_rop` | Redshift Render. |
+| `Review` | `review` | Review with OpenGL ROP. |
+| `Static Mesh (FBX)` | `staticMesh` | Static Meshes as FBX. |
+| `USD (experimental)` | `usd` | Universal Scene Description. |
+| `USD Render (experimental)` | `usdrender` | USD Render. |
+| `VDB Cache` | `vbdcache` | OpenVDB from Geometry ROP. |
+| `VRay ROP` | `vray_rop` | VRay Render. |
+
+* **Product Type Label:** It's mostly used in AYON UI e.g. Publisher Tool.
+* **Product Type Name:** The name used in the background. AYON Settings expects you to use this name not the label.
```

Comment:
Is there a reason the table is duplicated?

Reply:
Oh, That's happened by accidents few minutes ago. 

Should be fixed here https://github.com/ynput/ayon-documentation/pull/163/commits/f1ff6c0850b51788750fb3ed6bbb8364fca48846

Thanks!